add_options.rb

```diff
@@ -41,4 +44,33 @@ def add_music_album
     print ENTR_MSG
     gets
   end
+
+  def add_game
+    puts 'Pleae enter the publish date of the game:'
+    publish_date = enter_date
+    puts 'Is it multi_player:'
+    multi_player = gets.chomp
+    puts 'The date of the last player:'
+    last_player_at = enter_date
+    @games << Game.new(publish_date, multi_player, last_player_at)
+    puts 'Game is added succussfully'
+  end
+
+  def enter_date
+    puts 'Year (yyyy): '
+    year = gets.chomp.to_i
+    puts 'Month (1-12): '
+    month = gets.chomp.to_i
+    puts 'Day (1-31): '
+    day = gets.chomp.to_i
+    begin
+      date = Date.new(year, month, day)
+    rescue StandardError
+      puts "Be sure that you entered the correct date format\n
+      Please repeat again:"
+      enter_date
+    else
+      date
+    end
+  end
```

Comment:
When I tried this method, it made me enter the date twice. I entered the exact same date on both occasions to make sure I didn't make a mistake. Please, review this method to confirm if you are having the same issue.

Reply:
Thanks for reviewing my pull request. Please can you send to me a screenshot of the issue? which date have you used?